Domains/4-Application/SynchroModeler/SynchroModeler.ecschema.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="SynchroModeler" alias="synchromod" version="01.00.00" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECSchemaReference name="BisCore" version="01.00.06" alias="bis" />
+
+    <ECEntityClass typeName="Operation" modifier="abstract">
+        <BaseClass>bis:InformationRecordElement</BaseClass>
+    </ECEntityClass>
+
+    <ECEntityClass typeName="SubdivisionOperation">
+        <BaseClass>Operation</BaseClass>
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="SubdivisionOperationRefersToOriginalElement" strength="referencing" strengthDirection="Forward" modifier="Sealed">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
```

Comment:
I understood the relationship between a SubdivisionOperation and the OriginalElement as an n-to-1 (i.e. foreign-key), rather than a n-to-n (i.e. link-table). If that is correct, such relationship should rather be introduced as a navigation property instead of a subclass of the bis:ElementRefersToElements relationship class.